.github/workflows/ci.yml

```diff
@@ -256,6 +259,19 @@ jobs:
       fail-fast: false
       matrix:
         mapdl-version: ['v21.1.1', 'v21.2.1', 'v22.1.0', 'v22.2.0', 'v22.2-ubuntu', 'v23.1.0']
```

Comment:
Don't you have a ``latest`` tag? That way it can be more obvious that you are running against  the latest version of MAPDL. If the ``latest`` tag is the same as v23.1.0, you might even consider adding the v23.1.0 tag to the excluded list.

I know that in such case, things will run twice for v23.1.0, but you are not being billed by the time consumed and it will make it more obvious that you only run against the latest version of MAPDL.

Also, if for some reason v23.1.0 are not aligned anymore because it has some bug-fixes or whatever, then you ensure that you are compatible with the latest MAPDL version.

Food for thought. But the PR LGTM!

Reply:
I do not have the latest tag I believe. But I will consider it although I dont like to run MAPDL twice. 

I will think about it. Because I have to spend sometime with the MAPDL docker image.